Build failures on i386 / 32bit

Hi,

building on i386 (and most likely also on other 32bit architectures - see https://wiki.debian.org/ArchitectureSpecificsMemo) fails with

```
In file included from tcp_buffer_tuner.c:8:
1424
./tcp_buffer_tuner.skel.nobtf.h:35889:17: error: static assertion failed due to requirement 'sizeof (s->bss->bpftune_init_net) == 8': unexpected size of 'bpftune_init_net'
1425
 35889 |         _Static_assert(sizeof(s->bss->bpftune_init_net) == 8, "unexpected size of 'bpftune_init_net'");
1426
       |                        ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
1427
./tcp_buffer_tuner.skel.nobtf.h:35889:50: note: expression evaluates to '4 == 8'
1428
 35889 |         _Static_assert(sizeof(s->bss->bpftune_init_net) == 8, "unexpected size of 'bpftune_init_net'");
1429
       |                        ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^~~~
1430
./tcp_buffer_tuner.skel.nobtf.h:35895:17: error: static assertion failed due to requirement 'sizeof (s->bss->nr_free_buffer_pages) == 8': unexpected size of 'nr_free_buffer_pages'
1431
 35895 |         _Static_assert(sizeof(s->bss->nr_free_buffer_pages) == 8, "unexpected size of 'nr_free_buffer_pages'");
1432
       |                        ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
1433
./tcp_buffer_tuner.skel.nobtf.h:35895:54: note: expression evaluates to '4 == 8'
1434
 35895 |         _Static_assert(sizeof(s->bss->nr_free_buffer_pages) == 8, "unexpected size of 'nr_free_buffer_pages'");
1435
       |                        ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^~~~
1436
6 errors generated.
```

https://salsa.debian.org/debian/bpftune/-/jobs/6742374

A quick look at the code shows more usage of long/unsigned long, so I think this needs a more general fix, like using uint*_t or similar.

Or do you not want to support 32bit architectures?

--- a/include/bpftune/bpftune.bpf.h
+++ b/include/bpftune/bpftune.bpf.h
@@ -205,7 +205,7 @@ unsigned int tuner_id;
 unsigned int strategy_id;
 unsigned int bpftune_pid;
 /* init_net value used for older kernels since __ksym does not work */
-unsigned long bpftune_init_net;
+unsigned long long bpftune_init_net;
 
 /* TCP buffer tuning */
 #ifndef SO_SNDBUF

--- a/src/tcp_buffer_tuner.bpf.c
+++ b/src/tcp_buffer_tuner.bpf.c
@@ -37,7 +37,7 @@ int kernel_page_size;
 int kernel_page_shift;
 int sk_mem_quantum;
 int sk_mem_quantum_shift;
-unsigned long nr_free_buffer_pages;
+unsigned long long nr_free_buffer_pages;
 
 #define tcp_tunable_corr(__id, __cookie, __newval, __tp, __field_type, __field)\
 	{								\

--- a/src/tcp_buffer_tuner.c
+++ b/src/tcp_buffer_tuner.c
@@ -89,7 +89,7 @@ int get_from_file(FILE *fp, const char *fmt, ...)
 	return ret;
 }
 
-long nr_free_buffer_pages(bool initial)
+long long nr_free_buffer_pages(bool initial)
 {
 	unsigned long nr_pages = 0;
 	char *mzone = "Normal";

